project/AppDependencies.scala

```diff
@@ -5,24 +5,23 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.13-play28",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "3.6.0-play-28",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.73.0",
+    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "3.32.0-play-28",
     "uk.gov.hmrc"       %% "domain"                         % "8.1.0-play-28",
```

Comment:
sometimes not used can be deleted if you can double check

Reply:
I think it's being used. It tried removing it and got some compilation errors